src/javascript/app_2/Modules/Trading/Components/Form/TradeParams/Duration/advanced_duration.jsx

```diff
@@ -134,35 +99,15 @@ const AdvancedDuration = ({
                 </Fragment> :
                 <Fragment>
                     <div className={endtime_container_class}>
-                        <DatePicker
-                            name='expiry_date'
-                            has_today_btn
-                            min_date={min_date_expiry}
-                            max_date={max_date_duration}
-                            start_date={start_date}
-                            onChange={onChange}
-                            value={expiry_date}
-                            is_read_only
-                            is_nativepicker={is_nativepicker}
+                        <TradingDatePicker
                             alignment='left'
```

Comment:
don't think `alignment` is needed here

Reply:
yes, i remove it, thank you @kellybinary